mini_pupper_control/mini_pupper_control/servo_interface.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# MIT License
+#
+# Copyright (c) 2022 MangDang
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# @Author  : Zhengxiao Han
+
+import numpy as np
+import rclpy
+from rclpy.node import Node
+from trajectory_msgs.msg import JointTrajectory
+from MangDang.mini_pupper.HardwareInterface import HardwareInterface
+
+
+class SERVO_INTERFACE(Node):
```

Comment:
How about using CamelCase?

https://peps.python.org/pep-0008/#class-names

```suggestion
class ServoInterface(Node):
```